src/autocomplete/index.ts

```diff
@@ -8,6 +8,9 @@ class Autocomplete {
   state: Autocomplete.State = <any>{
     onHover: (event: MouseEvent) => {
       const targets = this.activationTargets();
+      if (Array.from(targets).findIndex((element) => element === event.target) === this.state.selected) {
```

Comment:
`Array.from(targets).findIndex((element) => element === event.target` is being used twice, you can extract it